component/async/component.go

```diff
@@ -168,38 +168,32 @@ func (c *Component) processing(ctx context.Context) error {
 		return fmt.Errorf("failed to get consumer channels: %w", err)
 	}
 
-	failCh := make(chan error)
-
-	go func() {
-		for {
-			select {
-			case <-ctx.Done():
-				log.Info("closing consumer")
-				failCh <- cns.Close()
-			case msg := <-chMsg:
-				log.Debug("New message from consumer arrived")
-				c.processMessage(msg, failCh)
-			case errMsg := <-chErr:
-				failCh <- fmt.Errorf("an error occurred during message consumption: %w", errMsg)
-				return
+	for {
+		select {
+		case msg := <-chMsg:
+			log.Debug("New message from consumer arrived")
```

Comment:
maybe make this line too start with a lowercase letter and perhaps rephrase it to `consumer received a new message`. Also since it it the first start of log correlation in the consumer, perhaps `log.FromContext(msg.Context()).Debug()`?

Reply:
i agree

Reply:
Sure, this was pre-existing code unrelated to this PR but I can improve it.

Shall I also add the message with `.Sub()` for structured logging purposes?

Reply:
There is no need for this.

Reply:
Changed.

Reply:
I do not see the change actually... ;)

Reply:
It's here: https://github.com/beatlabs/patron/pull/205/files#diff-6c64b2e2b8b9d28dcc79aa7f5ec20954R174

Reply:
I will add the context too.

Reply:
you forgot to `log.FromContext(msg.Context()).Debug(...)`

Reply:
Done.

Reply:
Sorry, but you changed all logging... The only log that needed to be chnaged was the mentioned one...
There is no context need for the others.

Reply:
@mantzas sure, I changed them back.

I fixed a typo in the README as well.